### PR TITLE
Jpk/navigation

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,23 +1,63 @@
...
+
+  state = {
+    fontLoaded: false,
+  };
+
+  async componentDidMount() {
+    await Font.loadAsync({
+      'Montserrat-Light': require('./Fonts/Montserrat-Light.ttf'),
+      'Montserrat-SemiBold': require('./Fonts/Montserrat-SemiBold.ttf'),
+      'Montserrat-Medium': require('./Fonts/Montserrat-Medium.ttf'),
```
Comment (srkirkland) [inline App.js]: @jpknoll guess you are going to have to get the UCD font eh? :)

File: Containers/AboutScreen.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {
+  Image,
+  Text,
+  View,
+} from 'react-native'
+import { connect } from 'react-redux'
+import { Images } from '../Themes'
+import styles from './Styles/AboutScreenStyle'
```
Comment (srkirkland) [inline Containers/AboutScreen.js]: @jpknoll ugh getting flashbacks to ucsd with all these external styles

File: Containers/AboutScreen.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return {
+  }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+  }
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(AboutScreen)
```
Comment (srkirkland) [inline Containers/AboutScreen.js]: @jpknoll no reason to use redux at all on this screen, and in fact no reason to make it a component at all.  Pure function would do the job

Comment (jpknoll) [reply Containers/AboutScreen.js]: @srkirkland The original version was mapping a datetime redux to this screen, but yeah, it should be a simple component at least.

**Pull request merged**